### ISSUE TITLE
AS3 doesn't process monitor with line: "receive": "\\{status\\\" \\:\\\"UP\\\\"l (lonely '{')

### ISSUE THREAD

**Issue by p-semenov-f5** [F5Networks/f5-appsvcs-extension#513]
### Environment
 * Application Services Version: 3.31
 * BIG-IP Version: 16.1.0

### Summary
We have correct config for monitors:
```
ltm monitor https /Common/good_monitor {
    recv "\{\"status\"\:\"UP\"\}"
    ssl-profile /Common/serverssl
}
ltm monitor https /Common/good_monitor_2 {
    recv "\{status\" \:\"UP\\"
    ssl-profile /Common/serverssl
}
```
It can be checked on big-ip via "tmsh load sys config from-terminal merge"

I can convert that config to AS3 declaration via ACC without any issues. In result we have AS3 (look at step #1). And it can't be post.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "ADC",
    "schemaVersion": "3.31.0",
    "id": "urn:uuid:eb62db06-20bf-43d4-be5b-80440eef3348",
    "label": "Converted Declaration",
    "remark": "Auto-generated by AS3 Config Converter",
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared",
            "good_monitor": {
                "receive": "\\{\\\"status\\\"\\:\\\"UP\\\"\\}",
                "clientTLS": {
                    "bigip": "/Common/serverssl"
                },
                "class": "Monitor",
                "monitorType": "https",
                "transparent": false,
                "reverse": false
            },
            "good_monitor_2": {
                "receive": "\\{status\\\" \\:\\\"UP\\\\",
                "clientTLS": {
                    "bigip": "/Common/serverssl"
                },
                "class": "Monitor",
                "monitorType": "https",
                "transparent": false,
                "reverse": false
            }
        }
    }
}
```

2. Observe the following error response:
```json
        {
            "code": 400,
            "message": "declaration failed",
            "response": "POST http://admin:XXXXXX@localhost:8100/mgmt/tm/sys/config merge cli script response=400 body={\"code\":400,\"message\":\"incomplete command\",\"errorStack\":[],\"apiError\":26214401}",
            "host": "localhost",
            "tenant": "AS3_Tenant",
            "runTime": 1216
        }
```

### Expected Behavior
Declaration should be valid. Or provide right decision to convert right config line:
```
    recv "\{status\" \:\"UP\\"
```
### Actual Behavior
AS3 doesn't support recv line from monitor config with only one bracket {. If you delete it, declaration will be valid.

### Additional info
https://support.f5.com/csp/article/K15977

**Comment by p-semenov-f5** [comment]
https://jira.pdsea.f5net.com/browse/CHARON-463

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-2781.

**Comment by mdditt2000** [comment]
thanks @dstokesf5

**Comment by dstokesf5** [comment]
This has been resolved in the 3.32.0 release.
